### PR TITLE
Add timezone selector

### LINKED ISSUE #5628: Time Zone Selection
### Description

Add time zone selector in footer

### Problem to be solved

Users may not wish to use their local time zone

### Proposed solution

Add drop down selector to footer

image

#### Additional info

Suggestion from Wicked Bitcoin

https://x.com/w_s_bitcoin/status/1855598476068827248

### PR DESCRIPTION
**Pull request by natsoni**
Fixes #5628

This PR replaces the fee rate unit selector with a timezone selector to allow users to choose a timezone other than their local one:

Example with local timezone being UTC+1: 

https://github.com/user-attachments/assets/e9227d46-7499-4ae4-aa8b-84995a64a7d4

The fee rate unit selector will probably be reintroduced later in a dedicated settings page.

### CODE REVIEW

**Comment by softsimon** [comment]
I think you should add the cities to make it more user friendly to find what you're looking for.

`UTC+1 - Stockholm`

**Comment by natsoni** [comment]
Now shows the timezone names:

Screenshot 2024-11-14 at 17 08 27

If the user's system timezone is not a standard one like CET or whatever: 

Screenshot 2024-11-14 at 17 13 46

**Comment by wiz** [comment]
@softsimon is this ok to merge?

File: frontend/src/app/app.constants.ts [hunk 1]
```diff
@@ -439,4 +439,39 @@ export const fiatCurrencies = {
...
+
+export interface Timezone {
+  offset: string;
+  name: string;
+}
+
+export const timezones: Timezone[] = [
+  { offset: '-12', name: 'Anywhere on Earth (AoE)' },
+  { offset: '-11', name: 'Samoa Standard Time (SST)' },
+  { offset: '-10', name: 'Hawaii-Aleutian Standard Time (HST)' },
```
Comment (wiz) [inline frontend/src/app/app.constants.ts:452]:
```suggestion
  { offset: '-10', name: 'Hawaii Standard Time (HST)' },
  ```

**Review by softsimon — APPROVED** [review]
tested ACK @ [a2bc6f5bb]

**Pull request merged**